model/relabel/relabel.go

```diff
@@ -142,17 +142,13 @@ func (c *Config) UnmarshalYAML(unmarshal func(interface{}) error) error {
 // Regexp encapsulates a regexp.Regexp and makes it YAML marshalable.
 type Regexp struct {
 	*regexp.Regexp
-	original string
```

Comment:
:+1:

Reply:
This could be done, but I think it also means that the regexp.Regexp methods are not embedded anymore.

So every time we want to eg. call MatchString we first need to convert back to `re := regexp.Regexp(cfg.Regexp)` and then call the pointer method MatchString on _that_ object. Do you think it's worth it?

Reply:
Yeah, that seems cumbersome (I just played around with that a bit), so maybe let's keep it like you have it now.